<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COMP</class>
 <widget class="QWidget" name="COMP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>404</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="devilCount">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0/8</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="IconListWidget" name="devilList" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="fuseButton">
       <property name="text">
        <string>Fuse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="propButton">
       <property name="text">
        <string>Properties</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dismissButton">
       <property name="text">
        <string>Dismiss</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="duplicateButton">
       <property name="text">
        <string>Duplicate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="extractButton">
       <property name="text">
        <string>Extract</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IconListWidget</class>
   <extends>QWidget</extends>
   <header>IconListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
